dbms/src/Flash/Mpp/tests/gtest_mpptunnel.cpp

```diff
@@ -214,12 +142,140 @@ class MockAsyncCallData : public IAsyncCallData
     bool write_failed = false;
 };
 
+class MockExchangeReceiver
+{
+public:
+    MockExchangeReceiver(Int32 conn_num, Int32 output_stream_count_, bool enable_fine_grained_shuffle_)
```

Comment:
Please simplify this test class
It looks like both `memory tracker`, `waitAllConnectionDone`, `connectionDone`, and etc are useless

Reply:
> Please simplify this test class It looks like both `memory tracker`, `waitAllConnectionDone`, `connectionDone`, and etc are useless

- `memory tracker` is necessary as `LocalTunnel` needs this or it will execute with a nullptr.

I will delete `waitAllConnectionDone` and `connectionDone`.

Reply:
I think the memory tracker is nullptr is fine here.